lab27/exercise2/src/starWarsDisplay.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react';
+import $ from 'jquery';
```

Comment:
jQurey is far from being the best solution for HTTP calls.
Axios would be a much better choice. It is smaller and has much more flexibility.
Also, in general, it is not a good idea to import a whole library for just one method that you use. They probably have a way to import single functions.